apps/client/src/components/FreeSemesterForm/index.tsx

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable @rushstack/no-new-null */
+
+import { UseFormRegister } from 'react-hook-form';
+
+import { SelectIcon } from 'client/assets';
+import { scoreArray } from 'client/constants';
+import { cn } from 'client/lib/utils';
+import { ScoreFormType, SemesterId, SemesterType } from 'client/types';
```

Comment:
type을 import할 때 앞에 type 명시해주세요

```suggestion
import type { ScoreFormType, SemesterId, SemesterType } from 'client/types';
```

Reply:
피드백 감사합니다 [541fd76](https://github.com/themoment-team/hellogsm-front-24/pull/8/commits/541fd7625861a27dabff58766800eebc7d53abd3)